src/plugins/intel_cpu/src/nodes/bin_conv.cpp

```diff
@@ -786,7 +786,7 @@ struct jit_uni_bin_conv_kernel_f32 : public jit_uni_bin_conv_kernel, public jit_
                             uni_vmovups(ptr[reg_output + o_off * jcp_.typesize_out], vmm_dst | ktail_mask);
                         } else {
                             for (int oc = 0; oc < tail_size; oc++) {
-                                size_t o_off;
+                                size_t o_off = 0;
```

Comment:
This is to just consider and check a rule.
This is like false positive as variable is set in both cases.
Leaving not initialized values helps find some issue e.g. if the else case was empty static analysis should report error.

Reply:
In that particular case it just needs to be replaced with lambda. Done

> Leaving not initialized values helps find some issue e.g. if the else case was empty static analysis should report error.
Yes, and was reported here. Just fix of this issue was incorrect